src/components/Promotion/Comment/Comment.js

```diff
@@ -1,16 +1,59 @@
-import { useEffect, useRef, useState } from "react";
+import { useRef, useState } from "react";
 import { AiOutlineCheck, AiOutlineDelete, AiOutlineEdit } from "react-icons/ai";
 import styles from "../../../styles/Board/Promotion/Comment.module.scss";
-import AddComment from "components/Common/Comment/AddComment";
 import ReplyCommentContainer from "./ReplyCommentContainer";
 import ReplyAddComment from "./ReplyAddComment";
+import axios from "axios";
+import getToken from "utils/getToken";
 
 const Comment = ({ comment, postId, getData }) => {
   const [replyComment, setReplyComment] = useState(false);
+  const [isContentEditable, setIsContentEditable] = useState(false);
+  const descriptionDiv = useRef();
+  const token = getToken();
   const onClick = () => {
     setReplyComment(!replyComment);
   };
 
+  const onEdit = async () => {
+    const body = {
+      description: descriptionDiv.current.textContent,
+    };
+
+    if (isContentEditable) {
+      await axios
+        .put(
+          `http://3.36.72.145:8080/api/board/promotion/${postId}/${comment.no}`,
+          body,
+          {
+            headers: {
+              "x-auth-token": token,
+            },
+          }
+        )
```

Comment:
```suggestion
    if (isContentEditable) {
      await axios
        .put(
          `환경변수 적용/api/board/promotion/${postId}/${comment.no}`,
          {
      description: descriptionDiv.current.textContent,
    },
          {
            headers: {
              "x-auth-token": token,
            },
          }
        )
```

리팩터링2를 보니까 변수가 표현식과 다를바 없을때는 인라인을 해주라고 하네요
[변수 인라인하기](https://yujeongjeon.github.io/Chapter%2006%20-%20%EA%B8%B0%EB%B3%B8%EC%A0%81%EC%9D%B8%20%EB%A6%AC%ED%8C%A9%ED%84%B0%EB%A7%81/pages/6-4.html)